index.html

```diff
@@ -803,7 +806,9 @@ <h3>Verifiable Credentials</h3>
           <dt>Upgrade coupon for first class ticket</dt>
           <dd>Introduces commercial value in a verifiable credential </dd>
         </dl>
-        
+        <p>For details, refer to <a href='#example-verifable-credentials'>Example Verifiable
+        Credentials</a> in Appendix</p>
```

Comment:
This should be reworded.  Appendix B is "Example Verifiable Credentials".  Maybe:
```html
        <p>For details, refer to Appendix B: <a href='#example-verifable-credentials'>Example Verifiable
￼         Credentials</a>.</p>
```
Or use some auto linking magic.

Reply:
I didn't want to call out "Appendix B" in case we add a new appendix section.  ReSpec auto-numbers these, so I could change and the reference text would be out of sync.  Do you know if there is an way to accommodate that?  I don't know the auto linking magic :)

Reply:
The link provide direct reference.  Keeping it as is. Decided to pull in the PR as is.